src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -327,6 +327,9 @@ def avro(self, path: str) -> DataFrame:
         Args:
             path: The stage location of an AVRO file, or a stage location that has AVRO files.
 
+        Note:
+            When using :meth:`DataFrame.select`, quote the column names to select the desired columns.
```

Comment:
How about explaining to users why the quotes are needed?

Reply:
+1

Reply:
good point. added